mage_ai/services/k8s/job_manager.py

```diff
@@ -36,7 +35,7 @@ def __init__(
 
         self.pod_config = self.core_api_client.read_namespaced_pod(
             name=os.getenv(KUBE_POD_NAME_ENV_VAR),
-            namespace=os.getenv(KUBE_POD_NAMESPACE_ENV_VAR),
+            namespace=self.namespace,
```

Comment:
what's this for?

Reply:
This is a bit different.
The namespace passed to the JobManager `__init__` is the namespace for the job to be created.
The `os.getenv(KUBE_POD_NAMESPACE_ENV_VAR)` is the namespace of the current pod. The code here gets the config of the current pod.
The two values can be different.

I think the code can be updated to `os.getenv(KUBE_POD_NAMESPACE_ENV_VAR, self.namespace)` in case the  KUBE_POD_NAMESPACE_ENV_VAR environment variable is not set.